lib/utils/HttpClient.js

```diff
@@ -162,13 +163,13 @@ class HttpClient {
   }
 
   request(options, expectedStatusCode) {
-    const command = this.getCommand(options.url, options.method);
+    const command = this.getCommand(options, options.method);
     if (command) {
       logger.silly(`command config for ${options.url}:`, _.omit(command, 'Promise'));
       return Promise.try(() => command.execute(options, expectedStatusCode));
     }
     if (this.PARSED_URL_MAP[`${options.url}_${options.method}`] === undefined) {
-      logger.warn(`Circuit breaker not defined for : ${this.baseUrl}_${options.url}_${options.method}`);
+      logger.warn(`Circuit breaker not defined for : ${options.url} HTTP Method:${options.method}`);
```

Comment:
Shouldn't it be debug similar to https://github.com/SAP/service-fabrik-broker/pull/35/files#diff-b0429a6c736f8cb3d4e258a10501fb13R40

Reply:
No. I explictly kept this at warn as this helps in identifying URLs that are not configured for Circuit breaker easily. The line pointed by you gets invoked repeatedly every time so that was getting logged every time. However this line gets only logged only once. Post that `this.PARSED_URL_MAP` is updated with dummy object and this never gets logged. So having just one log message is OK.

Reply:
Understood. That should be fine.